### ISSUE TITLE
Radio button on top right is not changing theme in one go 

### ISSUE THREAD

**Issue by rudri182** [deepset-ai/haystack-website#341]
One feedback is that the radio button on the top right to change the page's theme is not changing the theme in one go. 

For the first time, it will be like the below:
![Screenshot from 2022-06-14 11-40-10](https://user-images.githubusercontent.com/31641813/173506374-3aa50069-bda6-4b3d-82aa-3dc22b3bda92.png)

After clicking it for the first time, it becomes like, 
![Screenshot from 2022-06-14 11-40-22](https://user-images.githubusercontent.com/31641813/173507057-63aa3ec1-3dc2-462f-b6b0-b5886fa95788.png)

And then it changes the theme to dark.
![Screenshot from 2022-06-14 11-40-35](https://user-images.githubusercontent.com/31641813/173507179-cb9aae3c-733b-476d-8b2f-0bf53412497c.png)
 
I hope this will help you to improve the website 🩹 💫

**Comment by vblagoje** [comment]
Yes, you are right @rudri182 , thanks for spotting and reporting this one.

**Comment by sjrl** [comment]
@rudri182 Could you check if this is still an issue for you? The button works for me in one go now.
